pubsub.go

```diff
@@ -61,19 +71,29 @@ func KeyToTopic(key string) string {
 // The constructor interface is complicated by the need to bootstrap the pubsub topic.
 // This could be greatly simplified if the pubsub implementation handled bootstrap itself
 func NewPubsubValueStore(ctx context.Context, host host.Host, cr routing.ContentRouting, ps *pubsub.PubSub, validator record.Validator) *PubsubValueStore {
-	return &PubsubValueStore{
+	psValueStore := &PubsubValueStore{
 		ctx: ctx,
 
-		ds:   dssync.MutexWrap(ds.NewMapDatastore()),
-		host: host, // needed for pubsub bootstrap
-		cr:   cr,   // needed for pubsub bootstrap
-		ps:   ps,
+		cr: cr, // needed for pubsub bootstrap
+
+		ds:                      dssync.MutexWrap(ds.NewMapDatastore()),
+		ps:                      ps,
+		host:                    host,
+		rebroadcastInitialDelay: 100 * time.Millisecond,
+		rebroadcastInterval:     time.Minute * 10,
```

Comment:
comically long names, compare with the surroundings.

Reply:
No longer than the similar heartbeats in gossipsub https://github.com/libp2p/go-libp2p-pubsub/blob/9f04364996b415168f0e0d7e9fc82272fbed4005/gossipsub.go#L31.

Reply:
Why do these have to be part of the type? They could be module level constants.

Note that the comically long names in gossipsub are in module level, where they have to be descriptive.

Reply:
I don't see why any of those variables should be at the module level, it makes it impossible to have multiple instances with different parameters. Global module definitions are strictly more restrictive with basically zero practical tradeoffs.

In a later PR we're going to have to have a discussion about changing the constructor on this struct anyway since when /libp2p/go-libp2p-pubsub#184 lands we will not need to pass in content routing anymore. We can discuss then whether rebroadcast delay/interval should be configurable via constructor.